src/main/java/ca/utoronto/fitbook/adapter/web/FollowController.java

```diff
@@ -9,16 +10,24 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 
+import javax.servlet.http.HttpSession;
+
 @Controller
 @RequiredArgsConstructor
 public class FollowController
 {
     private final FollowUseCase followUseCase;
 
     @PostMapping(path = "/follow")
-    String followUser(Model model, @RequestBody FollowCommand command) {
+    String followUser(Model model, HttpSession session, @RequestBody FollowRequestBody body) {
+        String userId = (String) session.getAttribute("userId");
+        if (userId == null)
+            throw new UnauthorizedUserException();
+        FollowCommand command = new FollowCommand(userId, body.getFolloweeId());
```

Comment:
Is it supposed to be followee?

Reply:
Yes the followee is the person being followed